app/views/partials/events/_card.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class='w-full flex flex-col gap-2'>
+  <div class='flex justify-between gap-10'>
+    <div>
+      <%= content_tag(:span, event.title, class: 'block') %>
+      <%= content_tag(:span, event.description, class: 'block') %>
+    </div>
+    <div>
+      <%= content_tag(:span, "from: #{event.start_date}", class: 'block') %>
+      <%= content_tag(:span, "to: #{event.end_date}", class: 'block') %>
+    </div>
+  </div>
+  <%= content_tag(:span, "--- #{event.creator.username} ---", class: 'block') %>
```

Comment:
because of this, you'll have a N+1 problem. How should you fix it?

Reply:
ah ok, that is because first, Rails is querying for all users, and for each, is querying the `creator` user,  right?

I think I can fix it at the controller by including the :creator in @events variable:

```ruby
def index
  @events = Event.all.includes(:creator)
end
```

Reply:
cool cool cool